src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -489,10 +489,11 @@
     <Compile Include="Bindings\BindingProviders\Rule.cs" />
     <Compile Include="Bindings\BindingProviders\FluentBindingProvider.cs" />
     <Compile Include="Bindings\BindingProviders\FilteringBindingProvider.cs" />
-    <Compile Include="Bindings\BindingProviders\FluidBindingProviderExtensions.cs" />
+    <Compile Include="Bindings\BindingProviders\FluentBindingProviderExtensions.cs" />
```

Comment:
Right. I have a correpsonding BOT change. Staging would be:
- update SDK
- update BOT 
- update SCript with new SDK + new Bot 

---
In reply to: [114213832](https://github.com/Azure/azure-webjobs-sdk/pull/1124#discussion_r114213832) [](ancestors = 114213832)